src/Tests.IntegrationTests/Helpers/RequestHelper.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Net;
+using System.Text;
+using System.Text.Json;
+using Tests.IntegrationTests.Models;
+
+namespace Tests.IntegrationTests.Helpers
+{
+    public static class RequestHelper
+    {
+        /// <summary>
+        /// Simple helper to handle http requests and deserialisation of result
+        /// </summary>
+        /// <typeparam name="T">Type that the result shoudl be deserialised to</typeparam>
+        /// <param name="httpClient">A HttpClient to use/reuse</param>
+        /// <param name="verb">Which Http verb to use</param>
+        /// <param name="uri">The Uri to send the request to</param>
+        /// <param name="uri">The Uri to send the request to</param>
+        /// <param name="apiKey">The value to send as x-api-key header</param>
+        /// <returns>Object representing deserialised result of type defined by T</returns>
+        /// <exception cref="NotImplementedException">If verb is not in expected range.</exception>
+        public static async Task<T> Request<T>(HttpClient httpClient, Verb verb, Uri uri, string apikey, string body = "")
```

Comment:
So that it is clear what the generic type is doing please do this:
```suggestion
        public static async Task<TResponse> Request<TResponse>(HttpClient httpClient, Verb verb, Uri uri, string apikey, string body = "")
```

Reply:
The suggestion does not compile. I have made the XML comment for `<typeparam name="T">` clearer to eliminate any confusion.

Reply:
As @decodingahmed suggested offline, it did not compile because it was used further down in the method, Have changed it to use `TResponse` now. Thanks for the suggestion.